### ISSUE TITLE
Confusion about texture loss

### ISSUE THREAD

**Issue by jiemojiemo** [aiff22/DPED#8]
the texture loss is defined as following:
```
loss_discrim = -tf.reduce_sum(discrim_target * tf.log(tf.clip_by_value(discrim_predictions, 1e-10, 1.0)))
loss_texture = -loss_discrim
```
in this case loss_texure is a negtive value, and I wonder if this is an bug. So I think the correct texture loss is :
```
loss_discrim = -tf.reduce_sum(discrim_target * tf.log(tf.clip_by_value(discrim_predictions, 1e-10, 1.0)))
loss_texture = loss_discrim
```

**Comment by aiff22** [comment]
Hi @jiemojiemo,

```loss_discrim``` defines discriminator loss function, while ```loss_texture``` is the loss of the enhancement network, which target is opposite to the discriminator's one. Thus, the sign of ```loss_texture``` is opposite to ```loss_discrim```, and its value is indeed negative.
